services/app-api/handlers/reports/create.ts

```diff
@@ -242,7 +248,12 @@ export const createReport = handler(
           workPlanMetadata
         ),
         reportYear,
-        reportPeriod: calculatePeriod(currentDate, workPlanMetadata),
+        reportPeriod: reportPeriod,
```

Comment:
tip: since the key and value here have the same name you can just write `reportPeriod,` like how the line above is just `reportYear,`!

Reply:
ohhh nice! that's good to know!